x-pack/plugins/embeddable_enhanced/public/embeddables/dynamic_action_storage.test.ts

```diff
@@ -91,24 +83,24 @@ describe('EmbeddableActionStorage', () => {
         action: {} as any,
```

Comment:
type `as unknown as SerializedAction`

Reply:
Done in 343e464a979251cae24177f87eb348c52ab673b0